lib/tools.js

```diff
@@ -8,51 +8,49 @@ const chalk = require('chalk')
 const warning = chalk.keyword('yellow')
 const info = chalk.keyword('green')
 
-let packages
-
-try {
-  const output = execSync('npx lerna ls --all --json')
-  packages = JSON.parse(output.toString()).map(
-    packageItem => `../packages/${packageItem.name}`
-  )
-} catch (error) {
-  console.log(warning('No packages found'))
-  process.exit(0)
+const pkgDirs = getPkgDirs()
+function getPkgDirs() {
+  try {
+    const packages = execSync('npx lerna ls --all --json')
+    return JSON.parse(packages.toString()).map(
+      ({ location }) => location
+    )
+  } catch (error) {
+    console.log(warning('No packages found'))
+    process.exit(0)
+  }
 }
 
 function cleanEnv() {
-  for (const pathDir of packages) {
-    const env = path.resolve(__dirname, `${pathDir}/.env`)
+  for (const pkgDir of pkgDirs) {
+    const env = `${pkgDir}/.env`
 
-    if (!fs.existsSync(env)) {
-      continue
-    }
+    if (!fs.existsSync(env)) continue
 
     try {
       fs.unlinkSync(env)
-      console.log(info(`Removed env on ${pathDir}`))
+      console.log(info(`Removed .env file at ${pkgDir}`))
     } catch (err) {
-      console.log(warning(`No env file on ${pathDir}`))
+      console.log(warning(`No .env file at ${pkgDir}`))
     }
   }
 }
 
 function createEnv() {
-  for (const pathDir of packages) {
-    const sample = path.resolve(__dirname, `${pathDir}/.env.sample`)
-    const env = path.resolve(__dirname, `${pathDir}/.env`)
+  for (const pkgDir of pkgDirs) {
+    const sample = `${pkgDir}/.env.sample`
 
-    if (!fs.existsSync(sample)) {
-      continue
-    }
+    if (!fs.existsSync(sample)) continue
```

Comment:
formatting: let's keep the curly braces for consistency

Reply:
done